lms/envs/devstack.py

```diff
@@ -215,6 +215,8 @@ def should_show_debug_toolbar(_):
 CORS_ORIGIN_WHITELIST = ()
 CORS_ORIGIN_ALLOW_ALL = True
 
+################################ CCX ###############################
+FEATURES['CUSTOM_COURSES_EDX'] = True
```

Comment:
Either that (since enabling CCX on devstack is logically its own thing and could merit its own PR).
Or, I'm thinking all these changes at the modulestore layer need to be in their own PR anyway.  That is, the introduction of make_course_usage_key.  So you can include it as part of that.